Resources/Prototypes/_Sunrise/Maps/Pools/default.yml

```
@@ -3,33 +3,5 @@
-  # SUNRISE-TODO: Старые-новые карты
-  #- Glacier
-  #- Shoukou
-  #- Hammurabi
-  #- Lighthouse
-  #- Angle
-  #- Pebble
-  #- Ovni
-  #- Kettle
-  #- Tortuga
-  #- Arena
+  maps: # Fire edit - все санрайзовские карты удалены в качестве оптимизации использования RAM
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)